src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1077,32 +1077,74 @@ private void terminateParticipant(
                 reason,
                 sendSessionTerminate));
 
+        List<Throwable> suppressedThrowables = new ArrayList<>();
```

Comment:
wow now this looks nasty

Reply:
I think those catch throwable are really ugly and it may be better to adjust the code that sends packet to do check for "is connected" and NOOP instead of failing (eventually warning that failed to cleanup)